py-polars/polars/internals/expr.py

```diff
@@ -926,6 +973,22 @@ def slice(self, offset: int | Expr, length: int | Expr) -> Expr:
             Start index.
         length
             Length of the slice.
+        Examples
+        --------
+
```

Comment:
I think you need a blank line before the Examples section, otherwise it won't render correctly.

Reply:
Thanks, done